source/base/mpi.cc

```diff
@@ -23,7 +23,7 @@
 #include <deal.II/lac/la_parallel_block_vector.h>
 #include <deal.II/base/multithread_info.h>
 
-#include <numeric> 
+#include <numeric>
```

Comment:
Not necessary, but also doesn't hurt :-)

Reply:
I get that after every change to the branch after running `astyle`. We should just fix it in one of the PRs. I guess we don't need an extra PR for that.